### PR TITLE
Corner radius of the lines for the BEMBoxTypeSquare box type

### PR DESCRIPTION
**Pull request by morozkin**
Hi @Boris-Em!

In my project i found it very useful to cutomize your default value of corner radius of the lines for the `BEMBoxTypeSquare` box. So i decided to create a new property for such customization and may be someone will also find this useful.

### CODE REVIEW

File: Classes/BEMCheckBox.m [hunk 1]
```diff
@@ -72,6 +72,7 @@ - (void)commonInit {
     _onCheckColor = [UIColor colorWithRed:0 green:122.0/255.0 blue:255/255 alpha:1];
     _tintColor = [UIColor lightGrayColor];
     _lineWidth = 2.0;
+	_squaredBoxCornerRadius = 3.0;
```
Comment (Boris-Em) [inline Classes/BEMCheckBox.m]: Nit: indentation.

File: Classes/BEMCheckBox.m [hunk 2]
```diff
@@ -167,6 +169,12 @@ - (void)setLineWidth:(CGFloat)lineWidth {
     [self reload];
 }
 
+- (void)setSquaredBoxCornerRadius:(CGFloat)squaredBoxCornerRadius {
+	_squaredBoxCornerRadius = squaredBoxCornerRadius;
```
Comment (Boris-Em) [inline Classes/BEMCheckBox.m]: Nit: indentation.

File: Classes/BEMCheckBox.m [hunk 3]
```diff
@@ -87,6 +88,7 @@ - (void)commonInit {
 - (void)initPathManager {
     _pathManager = [BEMPathManager new];
     _pathManager.lineWidth = _lineWidth;
+	_pathManager.squaredBoxCornedRadius = _squaredBoxCornerRadius;
```
Comment (Boris-Em) [inline Classes/BEMCheckBox.m]: Nit: indentation.

File: Classes/BEMCheckBox.h [hunk 4]
```diff
@@ -82,6 +82,10 @@ typedef NS_ENUM(NSInteger, BEMAnimationType) {
  */
 @property (nonatomic) IBInspectable CGFloat lineWidth;
 
+/** The corner radius of the squared box. Default to 3.0.
+ */
+@property (nonatomic) IBInspectable CGFloat squaredBoxCornerRadius;
```
Comment (Boris-Em) [inline Classes/BEMCheckBox.h]: I think that simply calling this property `conerRadius` would be less verbose.
We also need to clarify in the comments that `boxType` needs to be set to `BEMBoxTypeSquare` for this property to have any effect.

**Comment by morozkin** [comment]
Thanks for review @Boris-Em! I've fixed the problems you mentioned above.

**Comment by Boris-Em** [comment]
Thanks again for your work @morozkin!

**Pull request merged**